src/main/kotlin/repository/UniqueItem.kt

```diff
@@ -0,0 +1,8 @@
+package repository
+
+import java.util.*
+
+interface UniqueItem {
```

Comment:
Better to name "Identifiable", and remove getUniqueId method

Reply:
Having id in interface already means that we will be able to get value of id field

Reply:
![image](https://github.com/ponchik18/crypto/assets/153732658/e944d002-7271-4081-a949-25b2e33955d8)